src/credentials/sso/ssoSupport.ts

```diff
@@ -32,3 +34,17 @@ export function isSsoProfile(profile: Profile): boolean {
     }
     return false
 }
+
+export async function displaySsoProfileChosenMessage(): Promise<void> {
+    const signInInstructionMessage = `You have chosen an AWS Single Sign-On profile that requires authorization. `
+    await vscode.window.showInformationMessage(signInInstructionMessage, { modal: true })
```

Comment:
Is an await proper here? This can block execution of the rest of the workflow. Also, this is only being used in one place, you probably don't need the helper function.

Reply:
Function removed as it adds an extraneous modal.